CHANGELOG.md

```diff
@@ -12,6 +12,17 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🧬 A new experimental "disk monitor" feature was added that allows operators
+  to specify a maximum amount of disk space that can be used by VAST. Once
+  the database directory exceeds that limit, the oldest data will be erased
+  from the database to make space. The new command-line options `--disk-quota-high`,
+  `--disk-quota-low` and `--disk-quota-check-interval` were added for that purpose.
+  [#1103](https://github.com/tenzir/vast/pull/1103)
+
+- 🎁 New command-line options `--segments` and `--max-segment-size` were added
+  to control some details of the segments produced by the archive.
+  [#1103](https://github.com/tenzir/vast/pull/1103)
```

Comment:
```suggestion
- 🧬 A new *disk monitor* component can now monitor the database size and
  delete data that exceeds a specified threshold. Once VAST reaches the maximum
  amount of disk space, the disk monitor deletes the oldest data. The
  command-line options `--disk-quota-high`, `--disk-quota-low`, and
  `--disk-quota-check-interval` control the rotation behavior.
  [#1103](https://github.com/tenzir/vast/pull/1103)

- 🎁 The new command-line options `--segments` and `--max-segment-size` control
  how the archive generates segments.
  [#1103](https://github.com/tenzir/vast/pull/1103)
```

Reply:
I'd replace the second one with this:

> The new options `vast.segments` and `vast.max-segment-size` control how the archive generates segments.

The reason being that we do this everywhere else, and that the options include the hierarchy.